tests/test_filesize.py

```diff
@@ -0,0 +1,15 @@
+import os
+from os.path import getsize
+
+
+dir = ('safebrowsing')
+filenames = []
+for file in os.listdir(dir):
+        if os.path.isfile(os.path.join(dir, file)) and 'mozstd-trackwhite' in file:
```

Comment:
it seems like there will be at least one of these kinds of tests for each of the shavar files.  does the list of files already exist in a config somewhere that you might iterate over them one by one and check their filesize? (methinks that info is only in the testplan).
If I'm not mistaken, the challenge there will be that Francois had said certain file types should be <= a certain size and others would be different.  It may not hurt to add the filesizes to the prefs.ini, then ALL the info you need for your tests could be gotten from there.
Just my 2cents